### PR TITLE
Devolopment

### PR DESCRIPTION
**Pull request by PanahovKh**

### CODE REVIEW

**Comment by Musa-94** [comment]
> Why is the master branch not empty ????

> Why is the development branch merged into the master ????

File: index.js [hunk 1]
```diff
@@ -0,0 +1,238 @@
+// #1
+
+function getSum(a, b) {
+    let result = 0;
+
+    if ((a % 2 ) === 0) {
+       result = a * b
+    } else {
+    result = a + b
```
Comment (Musa-94) [inline index.js:9]: code style...

**Comment by PanahovKh** [comment]
А так не надо было?

чт, 20 февр. 2020 г., 20:53 Musa <notifications@github.com>:

File: index.js [hunk 2]
```diff
@@ -0,0 +1,238 @@
...
+}
+    return result 
+}
+console.log(getSum(2, 5))
+
+////#2
+
+function kordinat(x, y) {
```
Comment (Musa-94) [inline index.js:17]: the name of the function begins with the verb, then the noun.
`function getCoordinate(x, y) {}`

File: index.js [hunk 3]
```diff
@@ -0,0 +1,238 @@
...
+ s="точка находится в первой четверти"; }
+if(x < 0 && y > 0) {
+ s="точка находится во второй четверти";}
+if(x < 0 && y < 0) {
+ s="точка находится в третьей четверти";}
+if(x > 0 && y < 0) {
+ s="точка находится в четвертой четверти";}
+ 
+ 
+return s;
```
Comment (Musa-94) [inline index.js:30]: code style.....

File: index.js [hunk 4]
```diff
@@ -0,0 +1,238 @@
...
+    if(a > 0) {
+        result += a;
+    }
+    if(b > 0) {
+        result += b;
+    }
+    if(c > 0) {
+        result += c;
+    }
+    return result;
```
Comment (Musa-94) [inline index.js:49]: and again code style :\

File: index.js [hunk 5]
```diff
@@ -0,0 +1,238 @@
...
+    let multi = a * b * c;
+    let sum = a + b + c;
+    let result = 0;
+
+    if (multi > sum) {
+        result = multi + 3;
+    } else {
+        result = sum + 3;
+    }
+    return result;
```
Comment (Musa-94) [inline index.js:66]: return is always preceded by a paragraph !!!

File: index.js [hunk 6]
```diff
@@ -0,0 +1,238 @@
...
+    } else if (40 >= n <= 59) {
+        result = "D";
+    } else if ( 60 >= n <= 74) {
+        result = "C";
+    } else if (75 >=n <= 89) {
+        result = "B";
+    } else if ( 90 >= n <= 100) {
+        result = "A"
+    }
+ return  result;
```
Comment (Musa-94) [inline index.js:86]: return is always preceded by a paragraph !!!

**Comment by PanahovKh** [comment]
Можно удалить это?

чт, 20 февр. 2020 г., 20:53 Musa <notifications@github.com>:

File: index.js [hunk 7]
```diff
@@ -0,0 +1,238 @@
...
+    }
+    return result;
+}
+console.log(maxMultiple(2, 4, 6));
+
+//#5
+
+function findStudent (n){
+    if(0 > n <= 19) {
+        result = "F";
```
Comment (Musa-94) [inline index.js:74]: where is the result variable declared ???

File: index.js [hunk 8]
```diff
@@ -0,0 +1,238 @@
...
+//#Цыклы #1
+
+function oddNumbers () {
+    let sum = 0;
+    let quan = 0;
+    for (let i = 2; i < 99; i += 2) {
+        sum  += i;
+        quan ++;
+    }
+return `Summery is ${sum}, Quantiity ${quan}`;
```
Comment (Musa-94) [inline index.js:100]: code style.....

File: index.js [hunk 9]
```diff
@@ -0,0 +1,238 @@
...
+}
+console.log(oddNumbers());
+
+//#2
+
+function simpleElement (n) {
+
```
Comment (Musa-94) [inline index.js:107]: to whom left an empty line ????

File: index.js [hunk 10]
```diff
@@ -0,0 +1,238 @@
...
+//#2
+
+function simpleElement (n) {
+   
+    for (i = 2; i < n; i++) {
+        if (n % i !== 0) {
+            return (`simple`)  
+        } else {
+            return (`storng`)
```
Comment (Musa-94) [inline index.js:112]: it was possible to use a variable instead of two return.

File: index.js [hunk 11]
```diff
@@ -0,0 +1,238 @@
...
+function findMinElement(arr) {
+    let min = arr[0];
+
+    for( let i = 1; i < arr.length; i++ ) {
+        if( arr[i] < min ) {
+            min = arr[i];
+        } 
+    }
+
+    return min;
```
Comment (Musa-94) [inline index.js:141]: like)))

File: index.js [hunk 12]
```diff
@@ -0,0 +1,238 @@
...
+
+function findSumOddNumbers(arr) {
+    let sum = 0;
+
+    for( let i = 0; i < arr.length; i++ ) {
+        if( i % 2 !== 0) {
+            sum  = sum + arr[i];
+        }
+    }
+    return sum;
```
Comment (Musa-94) [inline index.js:205]: again code style...

File: index.js [hunk 13]
```diff
@@ -0,0 +1,238 @@
...
+}
+console.log(findSumOddNumbers([4, 6, 9, 3, 5, 0, 1]));
+
+//#6 
+
+function findReversArrays(arr) {
+    let revers = 0;
+
+    for( let i = arr.length - 1; i >= 0; i--) {
+
```
Comment (Musa-94) [inline index.js:215]: ,????????? o.O

File: index.js [hunk 14]
```diff
@@ -0,0 +1,238 @@
...
+function findQuantOddNumbers(arr) {
+    let quan = 0;
+
+    for( let i = 0; i < arr.length; i++ ) {
+        if( arr[i] % 2 !== 0) {
+            quan = quan + 1;
+        }
+    }
+    return quan;
```
Comment (Musa-94) [inline index.js:232]: xmmm... what do you think is the mistake ???

**Review by Musa-94 — COMMENTED** [review]
Unfortunately, not all tasks are done, so work on the code style, do not make such mistakes in the future, but not bad for a start!

**Pull request merged**